queries/ecma/highlights.scm

```diff
@@ -12,6 +12,10 @@
 (property_identifier) @property
 (shorthand_property_identifier) @property
 
+(variable_declarator
+  name: (object_pattern
+    (shorthand_property_identifier_pattern))) @property
```

Comment:
Looks like these should be variables, or from your example, constants.

Reply:
Ah, yes. Fixed 👍🏼